### DISCUSSION TITLE
Display prior message when replied-to

### DISCUSSION THREAD

**Discussion by dviner** [ReagentX/imessage-exporter#153] [Ideas]
In the html export, currently, if a message is replying to a prior message, the chat bubble shows the text "This message replies to an earlier message".  This is incredibly helpful when reading through a long chat history with lots of replies.

It would be even better if the text linked to the prior message, or displayed a translucent or miniature version of the message which is being replied to (akin to how imessage does it on an iphone).

Has this been attempted?

**Comment by ReagentX** [comment]
This is a really cool idea! I'm tracking it in #154.
